src/Compilers/Core/MSBuildTaskTests/GenerateMSBuildEditorConfigTests.cs

```diff
@@ -351,5 +367,12 @@ public void ItemMetadataPathIsAdjustedOnWindows()
 build_metadata.Compile.ToRetrieve = abc123
 ", result);
         }
+
+        private static ITaskItem GetTestFilePath([CallerMemberName] string callerName = "")
+        {
+            string executingLocation = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location)?.Replace('\\', '/') ?? string.Empty;
+            string path = $"{executingLocation}/{callerName}.GenerateMSBuildEditorConfig.editorconfig";
```

Comment:
>executingLocation [](http://example.com/codeflow?start=29&length=17)

It looks like the tests are writing to the test assembly directory. We should use a temp directory if possible. See uses of `Microsoft.CodeAnalysis.Test.Utilities.TempRoot` - for instance, tests that use `Temp.CreateDirectory()`.